cocos/audio/audio-source.ts

```diff
@@ -38,6 +38,11 @@ enum AudioSourceEventType {
     ENDED = 'ended',
 }
 
+class OperationInfo {
+    op: string = '';
+    params: any;
```

Comment:
```suggestion
    params: any[] | null;
```

Reply:
Currenttly, we don't need an array

Reply:
https://github.com/cocos/cocos-engine/blob/6adec7f7f6699ce30fc63c201236511eb1cde952/cocos/audio/audio-source.ts#L450

I saw here you use it as an array

Reply:
If params is set to any instead of any[], it will result in op being included in params and there will be a warning.